src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -71,13 +71,11 @@ public sealed class TokenCache : ITokenCacheInternal
         private TokenCacheCallback _userConfiguredAfterAccess;
         private TokenCacheCallback _userConfiguredBeforeWrite;
 
-        internal IServiceBundle ServiceBundle { get; private set; }
+        internal IServiceBundle ServiceBundle { get; }
```

Comment:
Small refactoring to make ServiceBundle readonly.

Reply:
Question, how will the property be set then if the property isn't initialized with "ServiceBundle { get; } = someBundle;". Serialization?

---
In reply to: [275567459](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1075#discussion_r275567459) [](ancestors = 275567459)

Reply:
@trwalke - If a property has no setter, it can still be "set" by the constructor, and only by the constructor. It is equivalent to a readonly property.